theorems/T000754.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: T000754
+if:
+  and:
+  - P000120: true
+  - P000041 : true
+then:
+  P000130: true
+refs:
+  - zb: "1458.54023"
+    name: Locally ordered topological spaces
+---
+
+We use the fact that {P133} + {P36} $\Rightarrow$ {P130}.
+It follows from theorems {T522}, {T203} and {T246}.
+
+Fix $x\in X$. By {P120} point $x$ has
+a neighbourhood that is {P133}, and by {P41}
+it has a (possibly smaller), connected neighbourhood.
+A connected subset of {P133} is orderable as well
+and hence is {P130}.
+Therefore $x$ has a local base of compact neighbourhoods.
+
+Cf. the proof of Lemma 3.1 b) in {{zb:1458.54023}}.
```

Comment:
```suggestion
We use the fact that {P133} + {P36} $\Rightarrow$ {P130},
which follows from the theorems {T522}, {T203} and {T246}.

Fix $x\in X$. By {P120} the point $x$ has
a neighbourhood that is {P133}, and by {P41}
it has a connected neighbourhood contained in the first one.
A connected subset of {P133} is orderable as well
and hence is {P130}.
Therefore $x$ has a local base of compact neighbourhoods.

See the proof of Lemma 3.1 b) in {{zb:1458.54023}}.
```
Needed to specify that the second nbhd was contained in the first one, so that it's connected inside LOTS.
Plus minor grammar fixes.

Reply:
Maybe, not sure.  It is not exactly a "meta-property" in the sense that the property is preserved by some basic topological operations.  It's more a combinations of operations, plus the original property, plus other properties (connecteed or compact).  It's certainly an interesting thing, but maybe not every topological fact needs to be in pi-base.  At least not yet?
  Should there be an issue to discuss the scope of meta-properties?  I wish @StevenClontz or @ccaruvana would be able to join that discussion.